src/web/pages/App.tsx

```diff
@@ -1,29 +1,16 @@
-import { useState } from 'react';
-import reactLogo from '../assets/react.svg';
-import viteLogo from '/vite.svg';
 import './App.css';
 
 function App() {
-  const [count, setCount] = useState(0);
 
   return (
     <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>count is {count}</button>
-        <p>
-          Edit <code>pages/App.tsx</code> and save to test HMR
+      <h1>FinOps toolkit </h1>
```

Comment:
Not sure if this is the right way to do a subheading, but copying this from the current site. Feel free to tweak as appropriate. Just wanted to bring in real content.
```suggestion
      <head>
        <h1>Kick start your FinOps efforts</h1>
        <h2>Automate and extend the Microsoft Cloud with starter kits, scripts, and advanced solutions to accelerate your FinOps journey.</h2>
      </head>
```